doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-        \begin{center}
-%            \resizebox{12cm}{8cm}{\includegraphics{img/simple-models.pdf}}
-            \scalebox{0.255}[0.4]{\includegraphics{img/simple-models.pdf}}
-            \caption{Simple models on \emph{austourists}.}
-            \label{img:simple-models}
-        \end{center}
-    \end{figure}
-
-    Exponential smoothing models are depicted in \ref{img:exp-smoothings}. For this particular series it is obvious that
-    seasonal model is the best describing modelled time series.
-
-    In the beginning it can be seen how is the seasonal model changing and learning the seasonal pattern from data.
-    Learning and adaptivity to new trends depends on smoothing parameters of the models. Higher values of parameters
-    allows the model quicker adapt to changes. Models with lower smoothing parameters are more robust to the changes.
-
-    \begin{figure}[H]
-        \begin{center}
-            \scalebox{0.255}[0.27]{\includegraphics{img/exp-smoothings.pdf}}
-            \caption{Exponential smoothing models on \emph{austourists}.}
-            \label{img:exp-smoothings}
-        \end{center}
-    \end{figure}
-
-    \begin{figure}[H]
-        \begin{center}
-            \scalebox{0.255}[0.27]{\includegraphics{img/arima-sarima.pdf}}
-            \caption{Arima models on \emph{austourists}.}
-            \label{img:arimas}
-        \end{center}
-    \end{figure}
-
-    ARIMA models are showed in \ref{img:arimas}. Non seasonal model is already capable of capturing some seasonal
-    patterns but it is more random whereas seasonal patterns should be of fixed period. Therefore seasonal ARIMA with
-    dedicated model for each period is more appropriate.
-
-    \begin{table}[h]
-        \begin{center}
-            \begin{tabular}{l|r|r|c}
-                    \textbf{Model} & \textbf{MSE} & \textbf{MAE} & \textbf{AIC} \\ \hline \hline
-                    Na\"{i}ve & 104.74 & 8.64 & \\
-                    Drift & 104.60 & 8.47 & \\
-                    Average & 79.91 & 7.14 & \\ \hline
-                    Simple exp. smoothing & 53.54 & 5.91 & 380.88 \\
-                    Double exp. smoothing & 44.41 & 5.28 & 375.90 \\
-                    Triple exp. smoothing & 5.34 & 1.71 & 282.73 \\ \hline
-                    Linear regression & 41.85 & 5.10 & 321.46 \\ \hline
-                    ARIMA(2, 1, 2) & 30.88 & 4.35 & 310.6 \\
-                    SARIMA(1, 0, 0)(1, 1, 0)[4] & 4.79 & 1.63 & 206.95 \\
-            \end{tabular}
-            \caption{Statistics of models.}
-            \label{tab:models-stat}
-        \end{center}
-    \end{table}
-
-    Statistics produced by each model are listed in table \ref{tab:models-stat}. Models with the lowest one
-    step ahead prediction error are seasonal ARIMA and triple exponential smoothing which is correct for this highly
-    seasonal time series.
-
-    As it was discussed in section \ref{sec:model-quality} for models comparison should be used AIC criterion. From the
```

Comment:
As it was discussed in section \ref{sec:model-quality} AIC criterion should be used for models comparison  .